courses/free-courses-bn.md

```diff
@@ -274,6 +274,7 @@ Online School
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - STUDY MART
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
https://www.youtube.com/playlist?list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ

escape the `|` or replace with `|`

Reply:
no ALL CAPS

Reply:
finished